### PR TITLE
argparse: PEP8 formatting

### PR DESCRIPTION
**Pull request by dxxb**
Fixes PEP8 compliance issues reported by https://pypi.python.org/pypi/pep8

### CODE REVIEW

File: argparse/argparse.py [hunk 1]
```diff
@@ -162,7 +168,7 @@ def _parse_args(self, args):
         # parse all args
         parsed_pos = False
         while args or not parsed_pos:
-            if args and args[0].startswith("-") and args[0] != "-" and args[0] != "--":
+            if args and args[0].startswith("-") and args[0] not in ["-", "--"]:
```
Comment (dpgeorge) [inline argparse/argparse.py]: Use a tuple, not a list.

**Comment by dpgeorge** [comment]
In general, lines up to about 90-ish characters are ok (by me at least, and by Raymond Hettinger https://www.youtube.com/watch?v=wf-BqAjZb8M).

**Comment by dxxb** [comment]
Everyone seems to have a different opinion about that and I am not about to start arguing about this ;) Shall I drop all line length related changes and keep the rest then?

File: argparse/argparse.py (hunk 1 above)
Comment (dxxb) [inline argparse/argparse.py]: Good point. However this hunk is going away because I understand you are OK with lines < 100 characters.

**Comment by dpgeorge** [comment]
> Shall I drop all line length related changes and keep the rest then?

Yes please.

**Comment by dxxb** [comment]
Done.

**Comment by pfalcon** [comment]
FYI, I've updated https://github.com/micropython/micropython-lib/wiki/ContributorGuidelines regarding PEP8/line length.

**Comment by dxxb** [comment]
Ack, thanks.

**Pull request merged**